prefab/README.md

```diff
@@ -0,0 +1,98 @@
+Prefab Samples
+==============
+The samples in this directory demonstrate how to create and consume C/C++
+dependencies with the Android Gradle Plugin. Dependencies are packaged within
+AARs using the [Prefab] package format. The Android Gradle Plugin natively
+supports producing and consuming these, so you do not need to worry about the
+details of the packaging format.
+
+The individual samples will explain in more detail, but the typical workflow for
+a consumer of a native dependency is:
+
+1. Enable the prefab build feature in your build.gradle file.
+1. Add the dependency to the dependencies block of your build.gradle file.
+1. Import the package in your CMakeLists.txt or Android.mk file.
+1. Link the dependencies to your libraries.
+
+Linking the dependency to your library will automatically make the headers
+available, link the required libraries, and include the dependencies you use in
+your APK or AAR.
+
+To produce an AAR that exposes C/C++ APIs:
+
+1. Enable the prefabPublishing build feature in your build.gradle file.
+1. Use android.prefab in your build.gradle file to declare the names of the
+libraries you wish to export and the headers that define their interface.
```

Comment:
Looks like the code formatting that was in the text I posted got dropped.

Reply:
added back some code format, modified the "Linking the .... " sentence clarify what it means ( it is hard one when suddenly dropping another "your APK or AAR" when this is mainly talking about how to use AAR, changed that one to "application", application in this sense could be APK or AAR, but this AAR has NOTHING to do with native dependency AAR)